Docs/ProjectSpec.md

```diff
@@ -776,7 +776,7 @@ The different actions share some properties:
 - [ ] **language**: **String** - `run` and `test` actions can define a language that is used for Application Language
 - [ ] **region**: **String** - `run` and `test` actions can define a language that is used for Application Region
 - [ ] **debugEnabled**: **Bool** - `run` and `test` actions can define a whether debugger should be used. This defaults to true.
-- [ ] **simulateLocation**: **[Simulate Location](#simulate-location)** - `run` action can define a simulated location
+- [ ] **simulateLocation**: **[Simulate Location](#simulate-location)** - `run` action can define a simulated location. See [Options](#options).
```

Comment:
Could you add a note down in `Simulate Location` that it will use `schemePathPrefix`. Also not sure what this `See Options` refers to?

Reply:
Good thinking! I meant to refer to `schemePathPrefix` within `Options`, but putting it in the `Simulate Location` section makes more sense